appstream.patch

```diff
@@ -0,0 +1,77 @@
+Date: Fri, 10 May 2024 16:40:35 +0000
+Subject: [PATCH 1/3] Add developer name.
+
+---
+ org.kde.sweeper.appdata.xml | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/org.kde.sweeper.appdata.xml b/org.kde.sweeper.appdata.xml
+index d2dcd0a..22f7499 100644
+--- a/org.kde.sweeper.appdata.xml
++++ b/org.kde.sweeper.appdata.xml
+@@ -3,6 +3,7 @@
+   <id>org.kde.sweeper.desktop</id>
+   <metadata_license>FSFAP</metadata_license>
+   <project_license>LGPL-2.1+</project_license>
++  <developer_name translate="no">KDE</developer_name>
+   <name>Sweeper</name>
+   <name xml:lang="ar">الكناس</name>
+   <name xml:lang="ast">Barrenderu</name>
+-- 
+GitLab
+
+
+Date: Fri, 10 May 2024 23:03:10 +0200
+Subject: [PATCH 2/3] Add launchable.
+
+---
+ org.kde.sweeper.appdata.xml | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/org.kde.sweeper.appdata.xml b/org.kde.sweeper.appdata.xml
+index 22f7499..f75b3fc 100644
+--- a/org.kde.sweeper.appdata.xml
++++ b/org.kde.sweeper.appdata.xml
+@@ -178,4 +178,5 @@
+     <release version="24.02.0" date="2024-02-28"/>
+     <release version="23.08.5" date="2024-02-15"/>
+   </releases>
++  <launchable type="desktop-id">org.kde.sweeper.desktop</launchable>
+ </component>
+-- 
+GitLab
+
+
+Date: Fri, 10 May 2024 20:49:00 +0000
+Subject: [PATCH 3/3] Fix AppStream ID.
+
+---
+ org.kde.sweeper.appdata.xml | 8 +++++++-
+ 1 file changed, 7 insertions(+), 1 deletion(-)
+
+diff --git a/org.kde.sweeper.appdata.xml b/org.kde.sweeper.appdata.xml
+index f75b3fc..d7c9e92 100644
+--- a/org.kde.sweeper.appdata.xml
++++ b/org.kde.sweeper.appdata.xml
+@@ -1,6 +1,6 @@
+ <?xml version="1.0" encoding="utf-8"?>
+ <component type="desktop">
+-  <id>org.kde.sweeper.desktop</id>
++  <id>org.kde.sweeper</id>
+   <metadata_license>FSFAP</metadata_license>
+   <project_license>LGPL-2.1+</project_license>
+   <developer_name translate="no">KDE</developer_name>
+@@ -179,4 +179,10 @@
+     <release version="23.08.5" date="2024-02-15"/>
+   </releases>
+   <launchable type="desktop-id">org.kde.sweeper.desktop</launchable>
++  <provides>
++    <id>org.kde.sweeper.desktop</id>
++  </provides>
```

Comment:
Looks like there is already a provides tag in metainfo file somewhere.

Reply:
Sorry, I am not very good at this patch juggling.